keps/sig-node/4816-dra-prioritized-list/README.md

```diff
@@ -0,0 +1,1116 @@
+Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+-->
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+The existing [integration tests for kube-scheduler which measure
+performance](https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf#readme)
+will be extended to cover the overheaad of running the additional logic to
+support the features in this KEP. These also serve as [correctness
+tests](https://github.com/kubernetes/kubernetes/commit/cecebe8ea2feee856bc7a62f4c16711ee8a5f5d9)
+as part of the normal Kubernetes "integration" jobs which cover [the dynamic
+resource
+controller](https://github.com/kubernetes/kubernetes/blob/294bde0079a0d56099cf8b8cf558e3ae7230de12/test/integration/scheduler_perf/util.go#L135-L139).
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+End-to-end testing depends on a working resource driver and a container runtime
+with CDI support. A [test
+driver](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/dra/test-driver)
+was developed as part of the overall DRA development effort. We will extend this
+test driver to enable support for alternative device requests and add tests to
+ensure they are handled by the scheduler as described in this KEP.
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+- Initial e2e tests completed and enabled
+
+#### Beta
+
+- Gather feedback
+- Additional tests are in Testgrid and linked in KEP
+
+#### GA
+
+- 3 examples of real-world usage
+- Allowing time for feedback
+
+### Upgrade / Downgrade Strategy
+
+Standard upgrade/downgrade strategies may be used, no special configuration
+changes are needed. There are no kubelet or DRA-driver changes for this feature,
+they are all local to the control plane.
+
+### Version Skew Strategy
+
+The proposed API change relaxes a `required` constraint on the
+`DeviceRequest.DeviceClassName` field. The `DeviceRequest` thus becomes a one-of
+that must have either the `DeviceClassName` or the `FirstAvailableOf` field
+populated.
+
+Older clients have been advised in the current implementation to check this
+field, even though it is required, and fail to allocate a claim that does not
+have the field set. This means that during rollout, if the API server has this
```

Comment:
@pohly I think this is similar to the discussion we had on the structured parameters KEP, and so I chose this behavior. WDYT?